foreshadow/utils/common.py

```diff
@@ -76,6 +76,7 @@ def get_transformer(class_name, source_lib=None):
                 "foreshadow.transformers.concrete",
                 "foreshadow.transformers.smart",
                 "foreshadow.transformers.core",
+                "foreshadow.cleaners.internals",
```

Comment:
Just want to flag this as something that won't be detected in the refactor

Reply:
(y) I think pycharm does but good to flag regardless